src/amazonqGumby/telemetry/codeTransformTelemetry.ts

```diff
@@ -63,4 +63,54 @@ export const JDKToTelemetryValue = (
     }
 }
 
+/**
```

Comment:
Tried to just add as much as I can from the oracle docs here -> https://docs.oracle.com/javase/specs/jvms/se21/html/jvms-4.html#jvms-4.1-200-B.2